.circleci/config.yml

```diff
@@ -44,6 +44,23 @@ commands:
       condition:
         type: string
     steps:
+      - run:
+          name: Ensure envsubst is installed
+          command: |
+            os_family="$(uname -s | tr '[:upper:]' '[:lower:]')"
```

Comment:
`uname` needs to used here with the shorter option because `osx` does not support the equivalent longest options (i.e. [uname --kernel-name](https://www.man7.org/linux/man-pages/man1/uname.1.html)). See: https://ss64.com/osx/uname.html